packages/lesswrong/components/posts/PostsPage/BestOfLessWrong/ReviewPillContainer.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Components, registerComponent } from '@/lib/vulcan-lib/components';
+import { AnalyticsContext } from '@/lib/analyticsEvents';
+import { useStyles, defineStyles } from '@/components/hooks/useStyles';
+import { useMulti } from '@/lib/crud/withMulti';
+import { HashLink } from '@/components/common/HashLink';
+import { hideScrollBars } from '@/themes/styleUtils';
+
+const styles = defineStyles("ReviewPillContainer", (theme: ThemeType) => ({ 
+  root: {
+    paddingTop: 24,
+    display: 'flex',
+    gap: '8px',
+    ...theme.typography.body2,
+    fontSize: '1.1rem',
```

Comment:
I don't remember the exact reason but iirc we disprefer `rem` (at least according to Habryka)